hypershift-operator/controllers/hostedcluster/hostedcluster_controller.go

```diff
@@ -1450,7 +1450,7 @@ func (r *HostedClusterReconciler) reconcileControlPlaneOperator(ctx context.Cont
 	}
 	controlPlaneOperatorDeployment := controlplaneoperator.OperatorDeployment(controlPlaneNamespace.Name)
 	_, err = createOrUpdate(ctx, r.Client, controlPlaneOperatorDeployment, func() error {
-		return reconcileControlPlaneOperatorDeployment(controlPlaneOperatorDeployment, hcluster, controlPlaneOperatorImage, r.AvailabilityProberImage, r.SocksProxyImage, r.TokenMinterImage, r.SetDefaultSecurityContext, controlPlaneOperatorServiceAccount, r.EnableCIDebugOutput, convertRegistryOverridesToCommandLineFlag(r.ReleaseProvider.GetRegistryOverrides()))
```

Comment:
The CPO from the payload uses the `Dockerfile.control-plane` which only adds the CPO binary. The idea was to make these a subcommand of the cpo, similiar to how the HCCO is already a subcommand: https://github.com/openshift/hypershift/blob/eb89a6ce1c655d3eba0b80a2884198c7488505fb/control-plane-operator/main.go#L54

Reply:
Ok that I don't necessarily know I'll be the best to build out then

Reply:
np, I can take take care later